### ISSUE TITLE
GAP - SpawnKerbal Player-Recovered Kerbals are still showing on roster after contract completion

### ISSUE THREAD

**Issue by inigma2** [inigmatus/GAP#100]
Raised: https://github.com/jrossignol/ContractConfigurator/issues/437

**Comment by jrossignol** [comment]
It's crazy that you're at issue #100 already.

**Comment by inigma2** [comment]
lol. that's only of the issues i've tracked. :)

**Comment by inigma2** [comment]
I may have to use RemoveKerbal as work around after CC 1.9.3 release. Might be a good idea to use it anyways for a proper cleanup of contracts.

**Comment by inigma2** [comment]
from:
http://forum.kerbalspaceprogram.com/index.php?/topic/129208-contract-pack-giving-aircraft-a-purpose-gap-101-beta-air-flights-tours-coast-guard-sts-jan-5-2016/&do=findComment&comment=2351927

So I just noticed something, everytime I complete the contract and inigma remains part of my crew roster, I've failed to complete the mission in the time frame given, think it's like 5 minutes? the contract still completes even though time expired. the first time I reported inigma being dead was the only time I completed the mission in less than 5 minutes. it seems that if you complete it fast enough inigma is removed as intended, but if you take too long, the contract still completes, but since the mission specs aren't met perfectly, inigma does not get removed.
